tests/unit/__snapshots__/storybook.spec.ts.snap

```diff
@@ -2206,11 +2206,7 @@ exports[`Storyshots MobilePrimitives/Numeric Response Block Existing Data For Te
                                           </div>
                                         </div>
                                          
-                                        <div
```

Comment:
this should have been done in other PR (which was already merged)

Reply:
@safydy-r  I think we don't want to remove this piece. But remove these features in `builder.config` and `appconfig` so that these buttons don't show up in the UI. This needs to be hidden in the community version, but seen when embedded in voto5. 
@bzabos  Can you confirm if my understanding is right?

Reply:
@poojahpatil90 , this is just a snapshot. The test found we shouldn't add call-to-record based on permission from `builder.config`, so that's why we won't have it in snapshot ;D